src/Nuget/BinaryParsers.nuspec

```diff
@@ -18,10 +18,10 @@
     <dependencies>
       <group targetFramework=".NETStandard2.1">
         <dependency id="Sarif.Driver" version="2.4.12" exclude="Build,Analyzers" />
-        <dependency id="ELFSharp" version="2.13.2" exclude="Build,Analyzers" />
```

Comment:
>ELFSharp [](http://example.com/codeflow?start=24&length=8)

This file was either missed, or on purpose have a specific old version stated here.
I think this file should be updated, let me know if wrong.